test/loaders/bitmapFontParser.js

```diff
@@ -370,6 +370,50 @@ describe('PIXI.loaders.bitmapFontParser', function ()
             done();
         });
     });
+
+    it('should split fonts if page IDs are in chronological order', function (done)
+    {
+        const loader = new PIXI.loaders.Loader();
+
+        loader.add(path.join(this.resources, 'split_font2.fnt'));
+        loader.load(() =>
+        {
+            const page0 = path.join(this.resources, 'split_font_ab.png');
+            const page1 = path.join(this.resources, 'split_font_cd.png');
+
+            expect(loader.resources[page0].metadata.pageFile).to.equal('split_font_ab.png');
+            expect(loader.resources[page1].metadata.pageFile).to.equal('split_font_cd.png');
+
+            const font = PIXI.extras.BitmapText.fonts.split_font2;
+            const charA = font.chars['A'.charCodeAt(0) || 65];
```

Comment:
Any particular reason to have the `|| 65` here? If `charCodeAt()` stopped working, then that would be a problem for BitmapText.

Reply:
Copy and pasted from another test, but I think it’s silly in this context. Will fix